po/com.vysp3r.ProtonPlus.pot

```diff
@@ -18,7 +18,7 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 
 #: src/Windows/Main.vala:85
-msgid "A simple Wine and Proton-based compatiblity tools manager for GNOME"
```

Comment:
You never touch the msgid part of a .po or .pot file. What you do is change the value in the code and then run the command to update the po files. 
For me to allow this PR I would need you to fix the type that should be located at 'src/Windows/Main.vala:85' and then run 'ninja com.vysp3r.ProtonPlus-update-po' which should update the po files the proper way.
This makes me think that I may want to create a section to explain how to translate my application.

Reply:
Oops, you're right! I have also forgot to actually fix the typo in the [Main.vala](https://github.com/Vysp3r/ProtonPlus/blob/main/src/Windows/Main.vala#L85) source code.

Can you merge only the second commit (that adds the Czech translation)? Thanks!

Reply:
On the other hand, running `ninja com.vysp3r.ProtonPlus-update-po` will make the msgstr for this string fuzzy in all po files which is not ideal since the content of the translation actually did not change (just a small typo present in the original string was fixed).